lib/grell/capybara_driver.rb

```diff
@@ -16,7 +16,7 @@ def setup_capybara
 
       # Capybara will not re-run the block if the driver name already exists, so the driver name
       # will have a time integer appended to ensure uniqueness.
-      driver_name = "poltergeist_crawler_#{Time.now.to_i}".to_sym
+      driver_name = "poltergeist_crawler_#{Time.now.to_f}".to_sym
```

Comment:
I think it's ok, because this [spec](https://github.com/mdsol/grell/blob/develop/spec/lib/capybara_driver_spec.rb) ensures that Ruby can handle this type of symbol as both identification and dupplication.